hack/update-import-aliases.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+
+kube::golang::verify_go_version
+
+cd "${KUBE_ROOT}"
+
+ret=0
+go run cmd/preferredimports/preferredimports.go --confirm "$@" || ret=$?
+if [[ $ret -ne 0 ]]; then
+  echo "!!! Please see hack/.import-aliases for the preferred aliases for imports." >&2
```

Comment:
This `confirm` operation is for updating go files based on hack/.import-aliases.
So if error happens here, that can be a different reason from these go files import unpreferred aliases.
Better to output different error message.